src/csm_testing/tests/sat_functional/bootprep/test_bootprep.py

```diff
@@ -305,7 +308,8 @@ def delete_vcs_repository(cls, repo_name) -> str:
             response = requests.delete(url, headers=headers)
 
             if response.status_code != 204:
```

Comment:
Is a 204 returned regardless of whether the repository exists? What kind of response do you get if you try to delete a repository that does not exist?

Reply:
This will be addressed in [CRAYSAT-2007](https://jira-pro.it.hpe.com:8443/browse/CRAYSAT-2007)